registries.conf

```diff
@@ -0,0 +1,20 @@
+# For more information on this configuration file, see registries.conf(5).
+#
+# Registries to search for images that are not fully-qualified.
+# i.e. foobar.com/my_image:latest vs my_image:latest
+[registries.search]
+registries = []
+
+# Registries that do not use TLS when pulling images.
```

Comment:
Same as in the man page, does this include untrusted TLS?

Reply:
fixed.